jelares/portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,20 @@
 .title {
-    background-image: linear-gradient(to right, var(--gradColor1), var(--gradColor2));
-    height:30px;
-    padding-top:30px;
-    padding-bottom:30px;
-    padding-left:15px;
-    margin-bottom:50px;
+  background-image: linear-gradient(
+    to right,
+    var(--gradColor1),
+    var(--gradColor2)
+  );
+  height: 30px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 15px;
+  margin-bottom: 50px;
```

Comment:
move margin above padding for alphabetical ordering?

Reply:
These are Jesus's files, which were accidentally committed. I'm planning on removing them from the commit when I recommit.

Reply:
Done